js/bitget.js

```diff
@@ -1111,6 +1122,293 @@ module.exports = class bitget extends Exchange {
         return result;
     }
 
+    async fetchDeposits (code = undefined, since = undefined, limit = undefined, params = {}) {
+        /**
+         * @method
+         * @name bitget#fetchDeposits
+         * @description fetch all deposits made to an account
+         * @url https://bitgetlimited.github.io/apidoc/en/spot/#get-deposit-list
+         * @param {string|undefined} code unified currency code
+         * @param {int} since the earliest time in ms to fetch deposits for
+         * @param {int|undefined} limit the maximum number of deposits structures to retrieve
+         * @param {object} params extra parameters specific to the bitget api endpoint
+         * @param {string|undefined} params.pageNo pageNo default 1
+         * @param {string|undefined} params.pageSize pageSize default 20. Max 100
+         * @returns {[object]} a list of [transaction structures]{@link https://docs.ccxt.com/en/latest/manual.html#transaction-structure}
+         */
+        await this.loadMarkets ();
+        const currency = this.currency (code);
+        if (since === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchDeposits() requires a `since` argument');
```

Comment:
let's assume a default value here instead of throwing if not provided, let's say something like 1-year back or something

```
since = this.milliseconds () - 31556952000 // 1yr
```